Doxyfile

```diff
@@ -0,0 +1,2781 @@
+# Doxyfile 1.9.6
```

Comment:
Is this something that could also live in "docs" directory?

Reply:
Following Metal on this as well, I can change it but maybe keeping it consistent across TT repos should be something to aim for